### ISSUE TITLE
[Chore] Insert Data Dynamically Using JS in 'Open Source Programs' Page

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#431]
## Category

- [ ] Resource Addition
- [ ] Documentation
- [x] Codebase
- [ ] User Interface
- [ ] Feature Addition

## Description

We need to insert data dynamically using JavaScript in the 'Open Source Programs' Page.

Guidelines:

- Ensure that the JavaScript file is named 'app.js'
- The file needs to be created in 'client/pages/open-source-programs'

**Comment by nipunh** [comment]
@Sriparno08 Can you please assign me this issue?

**Comment by Sriparno08** [comment]
Sure, @nipunh! You're assigned. 👍

**Comment by VJsharan** [comment]
am i allowed to do this only if im assigned?

**Comment by rnavaneeth992** [comment]
@Sriparno08 Assign me this issue

**Comment by Sriparno08** [comment]
Yes, @VJsharan and in this case, another contributor is already assigned to this issue.

Hey @rnavaneeth992, another contributor is already assigned to this issue.

**Comment by VJsharan** [comment]
ok got it

**Comment by rnavaneeth992** [comment]
@Sriparno08 bro they are taking too much time
assign to me, I will complete it fast

**Comment by Sriparno08** [comment]
Hey @nipunh, are you still working on this issue?

Note: If there's no response from your side within 2 days, you will be unassigned unfortunately.

Hey @rnavaneeth992, if there's no response from the assignee within 2 days, you will be assigned.

**Comment by nipunh** [comment]
> Hey @nipunh, are you still working on this issue?
> 
> Note: If there's no response from your side within 2 days, you will be unassigned unfortunately.
> 
> Hey @rnavaneeth992, if there's no response from the assignee within 2 days, you will be assigned.

Already working on it, will complete soon.
